### ISSUE TITLE
A11y_ML .NET Model Builder_ Create New Experiment_ Label in Name: Visible label and accessible name (Programmatically given name) are not same for "Experiment Name" edit field

### ISSUE THREAD

**Issue by SaisreeDandamudi** [dotnet/machinelearning-modelbuilder#1138]
“[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

### GitHubTags:
#A11yWCAG2.1; #A11yTCS; #Win32; #DesktopApp; #Feature-ML.NETModelBuilder-Win32-Nov2020; #A11ySev2; #Benchmark; #Visual Studio; #Label in Name; #WCAG2.5.3

### Environment Details:
Microsoft Visual Studio 2019 Preview Version 16.8.0 Preview 6.0

### Screen Readers:
Narrator; NVDA

### Repro Steps:

1.Open Visual Studio Enterprise 2019 Preview 
2. Create a new .Net Core Console App (C#) 
3. Tab to "Solution Explorer" and right click on the project (Shift + F10) and then from the expanded list Select "Add" and then from the list items select "Machine learning"
4. Tab till "Object Detection" scenario and select the scenario
5. Tab till "Set up work Space" button and press enter
6. Open Accessibility Insights for windows and Inspect name for "Experiment name" control

### Actual Result:
Visible label and accessible name (Programmatically given name) are not same for "Experiment Name" edit field

### Expected Result:
Visible label and accessible name (Programmatically given name) should be same

### User Impact:
Screen Reader users and users who use speech to text converter will face difficulty in using the application if visible name and accessible name are not same

### Attachment for Reference

![Screenshot (178)](https://user-images.githubusercontent.com/51435580/101470650-44e45d00-396c-11eb-8714-955b3fe54acc.png)

**Comment by SaisreeDandamudi** [comment]
Verified the bug in the latest build and the issue is fixed.
Please find the below attachment for reference
Microsoft Visual Studio Enterprise 2019 Preview
Version 16.9.0 Preview 4.0 [30917.4.main]
![Screenshot (63)](https://user-images.githubusercontent.com/51435580/108044232-048a1500-7068-11eb-8e32-986472bdbb55.png)

**Comment by poojapatil36** [comment]
#Github tag: #Closed;
